boreas/alivedetection.c

```diff
@@ -76,6 +76,25 @@ scan (alive_test_t alive_test)
   g_message ("Alive scan %s started: Target has %d hosts", scan_id,
              number_of_targets);
 
+  /* Check first if consider alive test method is set. In case there
+   * that there is no scan restrictions, no other test will be performed.
```

Comment:
```suggestion
  /* Check first if consider alive test method is set. In case that
   * there is no scan restrictions, no other test will be performed.
```